### PR TITLE
Add Encuentro and DeporTV

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds two channels from Argentina.

### CODE REVIEW

File: channels/ar.m3u [hunk 1]
```diff
@@ -136,3 +136,7 @@ http://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8
 http://streamspub.manasat.com:1935/tvar/tvmanaar2/playlist.m3u8
 #EXTINF:-1 tvg-id="TVPublica.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/0/0a/Logo_Televisi%C3%B3n_P%C3%BAblica_Argentina.png" group-title="General",TV Pública (TVP) (1080p) [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/user/TVPublicaArgentina/live
+#EXTINF:-1 tvg-id="Encuentro.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="General",Encuentro [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UC1zLDoKL-eKmd_K7qkUZ-ow/live
```
Comment (LaneSh4d0w) [inline channels/ar.m3u:140]: Add as alternative  https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UC1zLDoKL-eKmd_K7qkUZ-ow/live

File: channels/ar.m3u [hunk 2]
```diff
@@ -136,3 +136,7 @@ http://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="DeporTV.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="Sport",DeporTV [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSmh3DFxBwFurMttT60PQ1g/live
```
Comment (LaneSh4d0w) [inline channels/ar.m3u:142]: Add as alternative https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UCSmh3DFxBwFurMttT60PQ1g/live

File: channels/ar.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/ar.m3u:140]: Both links work from here.

File: channels/ar.m3u (hunk 2 above)
Comment (jkemp814) [reply channels/ar.m3u:142]: Both links work from here.

File: channels/ar.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/ar.m3u:140]: Added

File: channels/ar.m3u (hunk 2 above)
Comment (sguinetti) [reply channels/ar.m3u:144]: Added

File: channels/ar.m3u [hunk 3]
```diff
@@ -138,5 +138,9 @@ http://streamspub.manasat.com:1935/tvar/tvmanaar2/playlist.m3u8
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/user/TVPublicaArgentina/live
 #EXTINF:-1 tvg-id="Encuentro.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="General",Encuentro [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UC1zLDoKL-eKmd_K7qkUZ-ow/live
+#EXTINF:-1 tvg-id="Encuentro.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="General",Encuentro [Not 24/7]
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UCSmh3DFxBwFurMttT60PQ1g/live
 #EXTINF:-1 tvg-id="DeporTV.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="Sport",DeporTV [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCSmh3DFxBwFurMttT60PQ1g/live
+#EXTINF:-1 tvg-id="DeporTV.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="" group-title="Sport",DeporTV [Not 24/7]
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UCSmh3DFxBwFurMttT60PQ1g/live
```
Comment (jkemp814) [inline channels/ar.m3u:146]: Same link as above? (Encuentro)

Comment (sguinetti) [reply channels/ar.m3u:146]: It's my fault. I back to edit.

**Review by jkemp814 — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**